CHANGES.rst

```diff
@@ -19,6 +19,11 @@ flat_field
 - Modify the test_flatfield_step_interface unit test to prevent it from causing
   other tests to fail [#7752]
 
+general
+-------
+
+- Require minimum asdf version 2.14.4 [#7801]
```

Comment:
Should also include an entry for `assign_wcs`, indicating the change that was made.

Reply:
Thanks! I added an entry to that section.